mix.exs

```diff
@@ -3,35 +3,23 @@ defmodule TM.Mercury.Mixfile do
 
   def project do
     [app: :tm_mercury,
-     version: "0.3.0",
-     elixir: "~> 1.4.0",
+     version: "0.4.0-dev",
+     elixir: "~> 1.5.0",
```

Comment:
are we using anything in 1.5 that requires us to require such a strict elixir version?